browser/migrations/0021_auto_20180219_1955.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.1 on 2018-02-19 19:55
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
```

Comment:
the other three migrations seem to add an institution field, then alter and then delete it. so they are unnecessary migrations. Please remove them.

Reply:
Removed the unwanted migrations